### PR TITLE
Feature/holdings/111 store user holdings

### PR DESCRIPTION
**Pull request by Martinelli-89**
Added fetch call for user deposit during login. 

Trello: https://trello.com/c/63oK97Vk/111-create-api-call-to-store-users-holdings-in-the-database

### CODE REVIEW

File: src/utils/apiUtils.js [hunk 1]
```diff
@@ -36,4 +36,35 @@ export const storeUserDetails = async (userID, name) => {
...
+}
+
+export const getUserHoldings = async (userID) => {
+  const url= `https://money-app-api-oi3xuie5la-nw.a.run.app//quadra-money/v1/user-holding/${userID}`;
+  const res = await fetch(url);
+  const json = await res.json();
+  let totalDeposit=0;
+  json.forEach(deposit => {
+    totalDeposit += deposit.amount;
+    console.log(typeof(totalDeposit));
```
Comment (olirob93) [inline src/utils/apiUtils.js:66]: do we need this log anymore

File: src/utils/apiUtils.js [hunk 2]
```diff
@@ -36,4 +36,35 @@ export const storeUserDetails = async (userID, name) => {
...
+  })
+  console.log(totalDeposit);
```
Comment (olirob93) [inline src/utils/apiUtils.js:68]: do we need this log

**Review by olirob93 — APPROVED** [review]
looks good, I know you mentioned to me about the storeHoldings but always worth adding that sort of thing to the PR notes

**Review by olirob93 — APPROVED** [review]
all good

**Review by ChrisWakley — APPROVED** [review]

**Pull request merged**